packages/frontend/src/utils/wallet.js

```diff
@@ -149,6 +151,13 @@ class Wallet {
         this.accountId = localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID) || '';
     }
 
+    KEY_TYPES = {
```

Comment:
Missing `const`

Reply:
It was a class field but I think its more appropriate as a static variable anyway 👍

Reply:
Ah good point, I always forget this horrifying ball of concerns is a class 😅